modules/db/src/main/scala/gem/dao/package.scala

```diff
@@ -93,6 +93,10 @@ package object dao extends MoreTupleOps {
   implicit val OffsetQMeta: Meta[Offset.Q] =
     AngleMetaAsSignedArcseconds.xmap(Offset.Q(_), _.toAngle)
 
+  // Wavelength maps to an integer in angstroms
+  implicit val WavelengthMeta: Meta[Wavelength] =
+    Meta[Int].xmap(Wavelength.unsafeFromAngstroms, _.toAngstroms)
+
```

Comment:
I thought about that but it seemed to me that the current state of representing the central wavelength in um in some cases and nm in others was too confusing and thought it best to guarantee that when you see a wavelength column you always know it is angstroms.

`Angle` seemed different because there are various usages and no universally applicable natural unit.  Maybe I'm wrong though.